pkg/cmd/kafka/topic/update/update.go

```diff
@@ -302,7 +301,8 @@ func runCmd(opts *options) error {
 	}
 
 	opts.Logger.Info(opts.localizer.MustLocalize("kafka.topic.update.log.info.topicUpdated", topicNameTmplPair, kafkaNameTmplPair))
-	return dump.Formatted(opts.IO.Out, opts.outputFormat, response)
+
+	return nil
```

Comment:
The value of `--output` should be respected. The current default is JSON and we should still print to JSON when it is specified. We need to change the default to nothing, and only then do you not print the object.

Reply:
I have removed output from update commands in service registry.
Return status should be enough for scripts

Reply:
> I have removed output from update commands in service registry.
> Return status should be enough for scripts

So you are saying maybe we should not return any data, ever?

Reply:
I'm saying what I have done for registry is:
- Removed output flag for update (and other commands that do not expect to return anything)
- Always print some message out to user and scripts use return code

To formalize this as guideline:
```
Commands that are not expected to print out anything else than status messages do not need an output flag.
```

If guideline is to always have output flag then we would print some output as 
```json
{
"message":"successfully done something"
}
```

Reply:
registry does have output flag for update in places: https://github.com/redhat-developer/app-services-cli/blob/d7268802175750df88e3942889b5cd376ec91c21/pkg/cmd/registry/artifact/crud/update/update.go#L95

I think we may not need output at all. I will create a PR on top of this one to apply it globally.